resources/web/wwi/x3d_scene.js

```diff
@@ -345,6 +348,18 @@ X3dScene.prototype = {
     });
   },
 
+  _setupEnvironmentMap: function() {
+    var backgroundMap;
+    if (typeof this.scene.background !== 'undefined' && this.scene.background.constructor.name === 'CubeTexture')
+      backgroundMap = this.scene.background;
+    this.scene.traverse(function(child) {
+      if (child instanceof THREE.Mesh && child.material && child.material.constructor.name === 'MeshStandardMaterial') {
```

Comment:
Why not using `Mesh.isMesh` function instead of `instanceof`?
The rest of the code is currently using the `is<Type>` methods because personally I found that it makes the code more readable.
But apart from personal preferences it is important to be consistent and use the same method (whatever) everywhere.

Same for the material and background:  `MeshStandardMaterial.isMeshStandardMaterial` and `CubeTexture.isCubeTexture`.

`CubeTexture.isCubeTexture` is not documented but I just checked and it is set in the code: https://github.com/mrdoob/three.js/blob/master/src/textures/CubeTexture.js#L23
The `CubeTexture` type check would also have to be replaced in the rest of the code because I didn't found the `isCubeTexture` before.

Reply:
Thank you, I go ahead with your proposition.